src/config.ts

```diff
@@ -405,9 +407,10 @@ export function parseInput(session: Session, input: string, config: Config, over
       .split('\\{').map(s => s.replace(/\{/g, '(')).join('\\{')
       .split('\\}').map(s => s.replace(/\}/g, ')')).join('\\}')
   } else {
-    input = input
-      .split('\\(').map(s => s.replace(/\(/g, '{')).join('\\(')
-      .split('\\)').map(s => s.replace(/\)/g, '}')).join('\\)')
+    // emmmmm doesn't know why substituting curly brackets makes the picture ugly, so it's temporarily commented
+    // input = input
+    //   .split('\\(').map(s => s.replace(/\(/g, '{')).join('\\(')
+    //   .split('\\)').map(s => s.replace(/\)/g, '}')).join('\\)')
```

Comment:
What is going on with these lines, could you show me why isn't it working?

Reply:
# For example:
## commented:
  ![ac7f4927d4146f9d816f2181f9af1359](https://github.com/koishijs/novelai-bot/assets/109729945/b2dfb194-877e-40a7-9a01-144cb9c34c9d)
  ![315d7ecc66e2258410c6381e4cec1a3e](https://github.com/koishijs/novelai-bot/assets/109729945/28b973d6-1052-4302-a817-44490cc6783f)
  ![5b6e089d3b94b9883fd52104895d7d47](https://github.com/koishijs/novelai-bot/assets/109729945/92463370-be95-4aa8-b26f-cc842b26e597)
  ![4fd993bde4d150c7107f874c8ae17b1d](https://github.com/koishijs/novelai-bot/assets/109729945/03768b3e-1d7f-4296-8e86-60a99dc5c32b)
  ![ba74e1f51d0cf0a0bb09ba414831896a](https://github.com/koishijs/novelai-bot/assets/109729945/39555647-89c2-4844-9077-37769362bcec)
## uncommented
  ![image](https://github.com/koishijs/novelai-bot/assets/109729945/cd1fa63a-33d3-4917-9aba-5eaf4db99a95)
  ![image](https://github.com/koishijs/novelai-bot/assets/109729945/1cf9058c-5a55-465d-a495-f916887854d6)
  ![image](https://github.com/koishijs/novelai-bot/assets/109729945/03661072-fad5-43a6-aa8c-6c7ecbb496b6)
  ![image](https://github.com/koishijs/novelai-bot/assets/109729945/14ea6211-72f0-4147-84cc-b8dd6930852d)
  ![image](https://github.com/koishijs/novelai-bot/assets/109729945/b069cdd1-5f95-4110-8d69-45c7e2595c86)


Color performance is not as good in uncommented cases，probably my supervisor's feelings

Reply:
I think that is because [NovelAI doesn't support vector multiplier](https://docs.novelai.net/image/strengthening-weakening.html), but using the `{keyword}` and `[keyword]` to strengthen or weaken the vectors.
So what you're writing is `{solo:1.2}` rather than `{{{{solo}}}}`, will it causing NovelAI failed to parse your prompt? Can you try these prompt on their website with the same parameters see whether you'd get the same result?

Reply:
I see, it's working!
## with correct vectors：
![image](https://github.com/koishijs/novelai-bot/assets/109729945/87dfd115-693b-4a8d-a00e-0d18b35471a7)
![image](https://github.com/koishijs/novelai-bot/assets/109729945/3101184b-7282-4281-b7a7-e8bea9a31ee8)
## with wrong vector:
![image](https://github.com/koishijs/novelai-bot/assets/109729945/b93c6804-aab1-49e8-af2c-cb577c078b49)
![image](https://github.com/koishijs/novelai-bot/assets/109729945/fae6acd4-af2a-4ca7-b2a7-8cb0aa5140c1)